test/school_test.rb

```diff
@@ -45,11 +45,41 @@ def test_it_can_add_student_names
   end
 
   def test_it_can_calculate_end_time
-    skip
+
     school1 = School.new('9:00', 7)
     school2 = School.new('9:00', 3)
 
     assert_equal '16:00', school1.end_time
     assert_equal '12:00', school2.end_time
   end
+
+  #Iteration 3 Tests:
+  def test_is_full_time?
+    school = School.new('9:00', 7)
+    test = school.is_full_time?
+
+    assert_equal true, test
```

Comment:
When something could return either true or false, it is best to test both cases. So, I would make another school that is not full time and verify that ouutcome.